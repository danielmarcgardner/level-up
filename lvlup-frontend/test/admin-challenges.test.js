import AdminChallenges from '../src/components/Admin/admin-challenges/admin-challenges-main';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

test('AdminChallenges should render', () => {
  const adminchallenges = shallow(
    <AdminChallenges />,
  );
  expect(toJson(adminchallenges)).toMatchSnapshot();
});
