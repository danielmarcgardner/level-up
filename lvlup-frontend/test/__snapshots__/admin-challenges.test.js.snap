exports[`test AdminChallenges should render 1`] = `
<div
  className="lvl-table">
  <Connect(ChallengesTable) />
</div>
`;
